packages/nimble-components/src/drawer/styles.ts

```diff
@@ -121,17 +117,21 @@ export const styles = css`
             ease-in;
     }
 
-    :host([location='right']) dialog.closing .dialog-contents {
-        animation-direction: reverse;
-    }
-
     @media (prefers-reduced-motion) {
         :host([location='left']) dialog.animating .dialog-contents,
         :host([location='right']) dialog.animating .dialog-contents {
-            animation-duration: 1ms;
+            animation: ni-private-drawer-fade-in-keyframes ${largeDelay} ease-in;
         }
     }
 
+    :host([location='left']) dialog.closing .dialog-contents {
```

Comment:
By moving these two `.closing` styles below the `prefers-reduced-motion` styles, I avoided having to duplicate them within the `prefers-reduced-motion` styles. Otherwise, the `prefers-reduced-motion` styles take precedence, and the animation direction is not reversed.